src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -664,26 +665,37 @@ public String toVariableNameString(final Inventory i) {
 								"The first requires that the player is online and also accepts only part of the name, " +
 								"while the latter doesn't require that the player is online, but the player's name has to be entered exactly.")
 				.usage("")
-				.examples("")
-				.since("1.0")
+				.examples(
+					"set {_p} to \"Notch\" parsed as a player # returns <none> unless Notch is actually online",
+					"set {_p} to \"N\" parsed as a player # returns Notch if Notch is online because their name starts with 'N' (case insensitive) however, it would return nothing if no player whose name starts with 'N' is online."
+				).since("1.0")
 				.defaultExpression(new EventValueExpression<>(Player.class))
 				.after("string", "world")
 				.parser(new Parser<Player>() {
 					@Override
 					@Nullable
-					public Player parse(String s, ParseContext context) {
+					public Player parse(String string, ParseContext context) {
 						if (context == ParseContext.COMMAND) {
-							if (s.isEmpty())
+							if (string.isEmpty())
 								return null;
-							if (UUID_PATTERN.matcher(s).matches())
-								return Bukkit.getPlayer(UUID.fromString(s));
-							List<Player> ps = Bukkit.matchPlayer(s);
-							if (ps.size() == 1)
-								return ps.get(0);
-							if (ps.size() == 0)
-								Skript.error(String.format(Language.get("commands.no player starts with"), s));
+							if (UUID_PATTERN.matcher(string).matches())
+								return Bukkit.getPlayer(UUID.fromString(string));
+							String name = string.toLowerCase(Locale.ENGLISH);
```

Comment:
Use the case sensitive option here from the config.sk. As FranKusmiruk (Snow-Pyon) stated here https://github.com/SkriptLang/Skript/issues/3803

Reply:
Quoted from Snow:
> (although they should remain case insensitive)

They are case insensitive here regardless of the config option. Not sure what you mean by your comment.

Reply:
Wouldn't it be best to support the case sensitivity option?

Reply:
I am not against it but this wasn't the state before and this could be a breaking changes if someone has their case sensitive option to true then suddenly their code breaks.

More opinions on this would be helpful.